packages/next/build/webpack/plugins/functions-manifest-plugin.ts

```diff
@@ -59,6 +77,58 @@ export default class FunctionsManifestPlugin {
   }
 
   apply(compiler: webpack5.Compiler) {
+    const handler = (parser: webpack5.javascript.JavascriptParser) => {
+      parser.hooks.exportSpecifier.tap(
+        PLUGIN_NAME,
+        (statement: any, _identifierName: string, exportName: string) => {
+          const { declaration } = statement
+          if (exportName === 'config') {
+            const varDecl = declaration.declarations[0]
+            const init = varDecl.init.properties[0]
```

Comment:
Could the `runtime` config be any of the properties so we need to iterate over them and locate the correct one e.g. if the config is `{ unstable_includeFiles: [], runtime: 'nodejs' }`?